tests/e2e/e2e_test.go

```diff
@@ -31,33 +35,119 @@ var (
 
 	installCTK bool
 
-	image string
+	ImageRepo string
+	ImageTag  string
 
-	sshKey  string
-	sshUser string
-	host    string
-	sshPort string
-)
+	sshKey      string
+	sshUser     string
+	host        string
+	sshPort     string
+	cwd         string
+	packagePath string
```

Comment:
I don't believe these are used.

Reply:
Umm yeah, that was used in a previos iteration but I ended up removing the need for it, thanks, I have removed it